internal/app/machined/pkg/controllers/network/link_spec.go

```diff
@@ -403,7 +403,7 @@ func (ctrl *LinkSpecController) syncLink(ctx context.Context, r controller.Runti
 		}
 
 		// sync UP flag
-		existingUp := existing.Attributes.OperationalState == rtnetlink.OperStateUnknown || existing.Attributes.OperationalState == rtnetlink.OperStateUp
+		existingUp := existing.Flags&unix.IFF_UP == unix.IFF_UP
```

Comment:
sync UP flag, ignore OperationalState here (as oper state might be down for other reasons, we did what we could do - add the UP flag)